Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs

```diff
@@ -143,12 +154,33 @@ public static void PurgeMediaCacheDirectly(this UITestContext context, bool togg
         /// <c>Lombiq.Tests.UI.Shortcuts</c> enabled.
         /// </summary>
         public static void ExecuteRecipeDirectly(this UITestContext context, string recipeName) =>
-            context.GoToRelativeUrl("/Lombiq.Tests.UI.Shortcuts/Recipe/Execute?recipeName=" + recipeName);
+            GoTo<RecipeController>(
+                context,
+                nameof(RecipeController.Execute),
+                (nameof(recipeName), recipeName));
+
+        /// <summary>
+        /// Navigates to a page whose action method throws <see cref="InvalidOperationException"/>. This causes ASP.Net
+        /// Core to display an error page.
+        /// </summary>
+        public static void GoToErrorPageDirectly(this UITestContext context) => GoTo<ErrorController>(context);
+
+        private static void GoTo<TController>(
```

Comment:
Nice!

Reply:
Thanks. Are we interested in making a more general version for HL?

Reply:
We talked about having some kind of statically typed way of routing, I'd be happy to see it.

Reply:
Me too!

Reply:
I'd love to port it to O1 and use it in CARE.

Reply:
Will do, added it to my backlog: https://lombiq.atlassian.net/browse/OSOE-53